pkg/cmd/grafana-cli/runner/wireexts_oss.go

```diff
@@ -5,18 +5,22 @@ package runner
 
 import (
 	"github.com/google/wire"
+	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/registry"
 	"github.com/grafana/grafana/pkg/services/encryption"
 	"github.com/grafana/grafana/pkg/services/encryption/ossencryption"
 	"github.com/grafana/grafana/pkg/services/kmsproviders"
 	"github.com/grafana/grafana/pkg/services/kmsproviders/osskmsproviders"
+	"github.com/grafana/grafana/pkg/services/licensing"
 	"github.com/grafana/grafana/pkg/services/sqlstore/migrations"
 	"github.com/grafana/grafana/pkg/setting"
 )
 
 var wireExtsSet = wire.NewSet(
 	wireSet,
 	migrations.ProvideOSSMigrations,
+	licensing.ProvideService,
+	wire.Bind(new(models.Licensing), new(*licensing.OSSLicensingService)),
```

Comment:
Is this change related to the rest?

Reply:
yes -- required for the kmsproviders initialization